src/main/java/com/redislabs/redisai/Dag.java

```diff
@@ -0,0 +1,79 @@
+package com.redislabs.redisai;
+
+import java.util.ArrayList;
+import java.util.List;
+import redis.clients.jedis.util.SafeEncoder;
+
+public class Dag implements DagRunCommands {
+  List<List<byte[]>> commands;
+  List<Boolean> tensorgetflag;
+
+  /** Direct acyclic graph of operations to run within RedisAI */
+  public Dag() {
+    this.commands = new ArrayList<>();
+    this.tensorgetflag = new ArrayList<>();
+  }
+
+  protected List<?> processDagReply(List<?> reply) {
+    List<Object> outputList = new ArrayList<>(reply.size());
+    for (int i = 0; i < reply.size(); i++) {
+      if (this.tensorgetflag.get(i) == true) {
+        outputList.add(Tensor.createTensorFromRespReply((List<?>) reply.get(i)));
+      } else {
+        outputList.add(reply.get(i));
```

Comment:
`         outputList.add(reply.get(i)); `
What is the return value in this case?

Reply:
String "OK"